src/nc_stats.c

```diff
@@ -827,7 +828,9 @@ stats_listen(struct stats *st)
         log_error("socket failed: %s", strerror(errno));
         return NC_ERROR;
     }
-
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(3,9,0)
```

Comment:
@ton31337 twemproxy can run on mac or freebsd also. so I think you should check other os too not only linux :)